code/offline-transactions/durable-nonce/create-nonce-account/main.en.ts

```diff
@@ -0,0 +1,38 @@
+import { clusterApiUrl, Connection, Keypair, Transaction, NONCE_ACCOUNT_LENGTH, SystemProgram } from "@solana/web3.js";
+import * as bs58 from "bs58";
+
+(async () => {
+  // connection
+  const connection = new Connection(clusterApiUrl("devnet"), "confirmed");
+
+  // 5YNmS1R9nNSCDzb5a7mMJ1dwK9uHeAAF4CmPEwKgVWr8
+  const feePayer = Keypair.fromSecretKey(
+    bs58.decode("588FU4PktJWfGfxtzpAAXywSNt74AvtroVzGfKkVN1LwRuvHwKGr851uH8czM5qm4iqLbs1kKoMKtMJG4ATR7Ld2")
```

Comment:
Same goes for here with generate and airdrop

Reply:
updated at [a88b46c5](https://github.com/solana-dev-adv/solana-cookbook/pull/21/commits/a88b46c59dfe980b347d354a797ed17a5a92ea6b)